TVRemoteControl/Supporting Files/TVRemoteControl-Bridging-Header.h

```diff
@@ -0,0 +1,20 @@
+//
+//  TVRemoteControl-Bridging-Header.h
+//  TVRemoteControl
+//
+//  Created by Daian Aiziatov on 12/01/2019.
+//  Copyright © 2019 Daian Aiziatov. All rights reserved.
+//
+
+#ifndef TVRemoteControl_Bridging_Header_h
+#define TVRemoteControl_Bridging_Header_h
+
+#import "MMLANScanner.h"
+#import "LANProperties.h"
+#import "PingOperation.h"
+#import "MMLANScanner.h"
+#import "MACOperation.h"
+#import "MacFinder.h"
+#import "MMDevice.h"
```

Comment:
The biggest part of these imports is not used in the project. I tried to exclude them and everything works fine. Please remove redundant imports

```
#import "MMLANScanner.h"
//#import "LANProperties.h"
//#import "PingOperation.h"
#import "MMLANScanner.h"
//#import "MACOperation.h"
//#import "MacFinder.h"
#import "MMDevice.h"
```

Reply:
Removed.